### PR TITLE
Fix SplitMulti docs references

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/python/hail/methods/statgen.py [hunk 1]
```diff
@@ -1833,7 +1833,7 @@ def split_multi_hts(ds, keep_star=False, left_aligned=False, vep_root='vep'):
         PID: str,
       }
 
-    For other entry fields, use :class:`.SplitMulti`.
+    For other entry fields, use :func:`.split_multi`.
```
Comment (catoverdrive) [inline hail/python/hail/methods/statgen.py]: this isn't true anymore, I don't think.

Comment (tpoterba) [reply hail/python/hail/methods/statgen.py]: what isn't true?

Comment (tpoterba) [reply hail/python/hail/methods/statgen.py]: you mean it should be "annotate_entries"?

Comment (catoverdrive) [reply hail/python/hail/methods/statgen.py]: sorry--I meant that I think that line is from when SplitMulti was a class that could do things to the entries. Probably makes more sense to remove it?

Comment (catoverdrive) [reply hail/python/hail/methods/statgen.py]: yeah

**Pull request merged**